releasenotes/notes/apm-dd-service-5e1b767108439bf8.yaml

```
@@ -0,0 +1,4 @@
+features:
+  - |
+    APM: Passes Tracer Service from client generated stats to the backend.
+      See unified service tagging at https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging
```

```suggestion
    APM: Client generated stats now forward the main global service to the backend.
```
I don't think that link serves any purpose in the changelog. Also, I'm not exactly sure how this is of any concern to the user. Does this fix a bug? Does it add any new feature? That's what the change log should talk about... Otherwise, this may not be change log worthy...

---

You're right. I think it's not log worthy, we are just changing how we do things on our side.